source/manual/2nd-line.html.md

```diff
@@ -7,50 +7,116 @@ section: 2nd line
+
+- Second line, which includes data from our Origin health and Edge health dashboards
+- Sidekiq
+- Application deployment dashboards
+
+[Read more about Grafana](https://docs.publishing.service.gov.uk/manual/grafana.html).
 
 ### PagerDuty
 
-Some alerts are urgent enough to warrant immediate attention, such as parts of the site becoming unavailable or large quantities of error pages being served. We use [PagerDuty][] to notify the primary and secondary engineers on Technical 2nd Line during office hours (9:30am to 5:30pm), and on-call engineers outside of office hours.
+Some alerts are urgent enough to warrant immediate attention, such as parts of the site becoming unavailable or large quantities of error pages being served. We use [PagerDuty][] to notify the primary and secondary engineers on Technical 2nd Line during office hours (9:30am to 5:30pm), and on-call engineers outside of office hours. We carry out a [Pagerduty drill](https://docs.publishing.service.gov.uk/manual/pagerduty.html#pagerduty-drill) every Wednesday morning at 10am UTC.
 
 [Read more about PagerDuty](/manual/pagerduty.html).
 
-### Incidents
+## Incidents
 
-If there is a service outage or loss of functionality to a service (whether external or internal), or a security vulnerability is discovered, Technical 2nd Line will [declare an incident](/manual/incident-management-guidance.html).
+If there is a service outage or loss of functionality to a service (whether external or internal), or a security vulnerability is discovered, Technical 2nd Line will [declare an incident](/manual/incident-management-guidance.html) and write up an incident report. We normally review incidents on Mondays at 2-3pm.
 
 ## Zendesk
 
 [Zendesk][zendesk] is our support ticketing system. When not dealing with incidents and alerts, we should be working through Zendesk tickets.
 
 Read more about [processing Zendesk tickets on Technical 2nd Line](/manual/zendesk.html).
 
-You will likely need to use [Grafana](/manual/grafana.html) to investigate service issues.
+## 2nd Line Trello Board
+
+We use the [GOV.UK Technical 2nd Line Trello board][] to capture pieces of work 2nd Line are required to do, such as:
+
+- Leaver tickets
+- Setting up production access
+- Recording technical issues
+
+The board is reviewed during the weekly 2nd Line handover meeting where developers can talk the next team through any new cards and oustanding issues.
+
+It is your responsibility to help maintain this board so that it is up to date for the next 2nd Line team.
+
+At the start of your 2nd Line shift you should:
+
+- Read through the cards under [alerts which don't add value](https://trello.com/c/A3mKmh5s/583-this-column-is-to-record-the-same-alerts-are-coming-up-again-and-again-but-cant-action-when-it-spikes) so that you are aware of any noisy Icinga alerts that can be ignored during your shift
```

Comment:
We have recently archived this column so we can merge this bullet point with the one below.

Reply:
I don't think it needs merging, just deleting?